examples/llama/pippy_llama.py

```diff
@@ -0,0 +1,109 @@
+# Minimum effort to run this example:
+# $ torchrun --nproc-per-node 2 pippy_llama.py
+
+import argparse
+import os
+
+import torch
+import torch.distributed as dist
+
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from pippy.IR import Pipe, PipeSplitWrapper, annotate_split_points
+from pippy.PipelineStage import PipelineStage
+
+
```

Comment:
it would be helpful if we add a comment here on our splitting points policy, how we are cutting the model/ how assigning devices to stages.

Reply:
Done!